src/main/java/com/thealgorithms/datastructures/queues/README.md

```diff
@@ -9,15 +9,57 @@
 
 ## Declaration
 
-`Queue<Obj> queue = new PriorityQueue<Obj> ();` 
+`Queue<Obj> queue = new LinkedList<Obj> ();` 
```

Comment:
How do you assign LinkedList to a queue?

Reply:
I have implemented the queue using LinkedList class because the time complexities provided in table exactly match this implementation.
We can implement queue in both ways it depends .

Reply:
You just cannot assign LinkedList to a Queue-typed variable

Reply:
@siriak please review this about my changes
When you write `Queue<Obj> queue = new LinkedList<Obj>(); `, you're using polymorphism. This means you're creating an instance of the LinkedList class, but you're treating it as if it's a Queue. This is possible because LinkedList is a subtype of Queue.
- If it's not okay then I can revert my change about implementing queue using LinkedList .

Reply:
Why do you think that LinkedList is a subtype of Queue? They are not related

Reply:
@siriak my bad I will rethink on it . I have reverted my change and requested to merge the pull request.